odoo/addons/base/views/res_company_views.xml

```diff
@@ -88,6 +88,16 @@
                                         <field name="phone"/>
                                     </div>
                                 </div>
+                                <div t-else="">
```

Comment:
event the original looks broken... for 1-2 years

`t-if="record.phone.value"` ... `<field name="email"/>` 🤔